pkgs/by-name/ca/cacert/setup-hook.sh

```diff
@@ -1,4 +1,8 @@
 export NIX_SSL_CERT_FILE="${NIX_SSL_CERT_FILE:-@out@/etc/ssl/certs/ca-bundle.crt}"
+if test ! -r "$NIX_SSL_CERT_FILE" ]; then
```

Comment:
This syntax seems weird.
Isn't it supposed to be `[ ! -r filename ]`?

Reply:
Oops yeah that syntax is a bit weird, fixed 👍